client/src/containers/ContactUsPage/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faTwitter,
+  faInstagram,
+  faFacebookSquare,
+} from '@fortawesome/free-brands-svg-icons';
+import contactPhoto from '../../assets/images/Group 3036.svg';
+
+import './style.css';
+
+const ContactUS = () => {
+  return (
+    <div className="contact-container">
+      <div className="content-container">
+        <div className="info-container">
```

Comment:
You might not need all of these nested divs - I wonder if you could remove `info-container` and just move its styling into `content-container`, to reduce a little bit of the nesting.